news/_posts/releases/2020-11-03-prestashop-1-7-7-0-rc-release.md

```diff
@@ -0,0 +1,131 @@
+- [Fix the impossibility to create an order when the address is considered as soft deleted](https://github.com/PrestaShop/PrestaShop/issues/20666)
+- ["Send an email to the customer with the link to process the payment." doesn't work](https://github.com/PrestaShop/PrestaShop/issues/20687)
+- [Missing gift wrapping and recycled paper options in create order page](https://github.com/PrestaShop/PrestaShop/issues/21300)
+- [In order detail page, when adding a discount amount with a different currency than the default one, an incorrect value is displayed and an incorrect amount is calculated](https://github.com/PrestaShop/PrestaShop/issues/21323)
+- [An exception is displayed when removing a discount with free gift from an order](https://github.com/PrestaShop/PrestaShop/issues/21438)
+- [Incorrect Product's total when adding a product with a discount gift or changing the Shipping](https://github.com/PrestaShop/PrestaShop/issues/21566)
+- [Incorrect total when updating a quantity's product](https://github.com/PrestaShop/PrestaShop/issues/21659)
+
+**Back Office:**
+- [Ranges are asked while they shouldn't during carrier edition](https://github.com/PrestaShop/PrestaShop/issues/20993)
+- [Following migration, $_GET identifier parameters are missing on "View {..} Page](https://github.com/PrestaShop/PrestaShop/issues/20934)
+- [Features page - an exception is displayed](https://github.com/PrestaShop/PrestaShop/issues/20777)
+- [Stats page - No module has been installed](https://github.com/PrestaShop/PrestaShop/issues/21186)
+- [An error is displayed when disabling or enabling maintenance](https://github.com/PrestaShop/PrestaShop/issues/21430)
+- [Hook displayBackOfficeTop not being called on Login BO Page](https://github.com/PrestaShop/PrestaShop/issues/21379)
+
+**Front Office:**
+- [Wrong cart rules discount when using alternative currency](https://github.com/PrestaShop/PrestaShop/issues/20594)
+- [Incorrect total with Free gift + Percent discount cart rule](https://github.com/PrestaShop/PrestaShop/issues/20690)
+- [When we have two currencies enabled - the Currency block and the language block are not working anymore](https://github.com/PrestaShop/PrestaShop/issues/20950)
+
+**For developers**
+- [Make PrestaShop compatible with composer 2.0](https://github.com/PrestaShop/PrestaShop/pull/21609)
+- [Update behat and gerkhin to allow debugging in PHPStorm](https://github.com/PrestaShop/PrestaShop/pull/21532)
+
+## Changelog
+
+151 pull requests have been merged since the beta 2 release and 30+ issues have been fixed. Read the [Changelog](https://github.com/PrestaShop/PrestaShop/releases) for details. 
+This brings us to more than 1400 merged pull requests for the 1.7.7.0 milestone, it is definitely our biggest yet.
+
+If you are looking for more details about all changes and new features expected in 1.7.7, the [1.7.7 beta version release note](https://build.prestashop.com/news/prestashop-1-7-7-0-beta-release/) is the perfect article for you to read!
+
```

Comment:
Add some infos about the tests here maybe? I don't know how to present it but here are the statistics:

```
Tests in 1.7.7:
UI (sanity + functional): 4307 (whole new campaigns)
Integration:
  - PHPUnit 55 tests (+3)
  - Behat 518 scenarios, 8838 steps (+297 scenarios, +6133 steps)
Unit: 1998 tests (+31)
```

And @SimonGrn suggested to integrate those two links if people want more info about UI tests:
- https://build.prestashop.com/news/the-new-prestashop-test-framework/
- https://build.prestashop.com/news/how-to-write-a-new-ui-test/